### PR TITLE
resolved all bytecompile warnings on emacs 27.1.90

### PR DESCRIPTION
**Pull request by emacs18**
Most changes are straight forward except two error message strings.  I took a guess at the format string arguments for two error messages.

### CODE REVIEW

File: org-db.el [hunk 1]
```diff
@@ -53,14 +53,12 @@
 
 (defcustom org-db-root "~/org-db/"
   "Root directory for db files."
+  :type 'string
```
Comment (jkitchin) [inline org-db.el:56]: This should be a 'directory probably.

File: org-db.el [hunk 2]
```diff
@@ -53,14 +53,12 @@
...
   :group 'org-db)
 
 (defcustom org-db-name "org-db.sqlite"
   "Name of the sqlite database file."
-  :group 'org-db)
-
-(defcustom org-db-index-content nil
-  "Controls if the content of headlines is saved."
+  :type 'string
```
Comment (jkitchin) [inline org-db.el:61]: This should be 'boolean.

File: org-db.el (hunk 1 above)
Comment (emacs18) [reply org-db.el]: Fixed

File: org-db.el (hunk 2 above)
Comment (emacs18) [reply org-db.el:61]: Fixed

**Comment by emacs18** [comment]
Thanks for merging.  I remembered why I removed org-db-index-content at first.  I don't think that variable is used anywhere.

**Comment by jkitchin** [comment]
That is true, I once tried to get full text search with org-db but it was too slow. I have removed this variable in a recent push today.

**Pull request merged**